lib/services/ios-project-service.ts

```diff
@@ -257,7 +275,15 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 <plist version="1.0">
 <dict>
     <key>method</key>
-    <string>${exportOptionsMethod}</string>
+	<string>${exportOptionsMethod}</string>`;
+	if (options && options.provision) {
+		plistTemplate += `    <key>provisioningProfiles</key>
+<dict>
```

Comment:
this plist seems duplicate of the one used on line 234, can we have it on a single place and reuse it